include/aos/common/crypto/mbedtls/cryptoprovider.hpp

```diff
@@ -157,6 +164,8 @@ class MbedTLSCryptoProvider : public aos::crypto::x509::ProviderItf {
     aos::Error SetCertificateAuthorityKeyIdentifier(mbedtls_x509write_cert& cert,
         const aos::crypto::x509::Certificate& templ, const aos::crypto::x509::Certificate& parent);
     aos::Error SetCertificateValidityPeriod(mbedtls_x509write_cert& cert, const aos::crypto::x509::Certificate& templ);
+
+    aos::StaticAllocator<cKeyAllocatorSize> mKeyAllocator;
```

Comment:
`mAllocator` it could be used for other purposes in the future.

Reply:
Done